orbit/cmd/orbit/orbit.go

```diff
@@ -762,104 +762,99 @@ func main() {
 				return fmt.Errorf("initializing token read writer: %w", err)
 			}
 
-			// note: the initial flags fetch above already populated the capabilities
-			// of the server based on the response header
-			if orbitClient.GetServerCapabilities().Has(fleet.CapabilityOrbitEndpoints) &&
-				orbitClient.GetServerCapabilities().Has(fleet.CapabilityTokenRotation) {
```

Comment:
The main change is this.

Due to the caching work added to `GetConfig` recently, the capabilities were not being set under certain circumstances (see the main issue on how to reproduce this).

Customers that are using a pretty old version of Fleet that doesn't support Fleet Deskop can run fleetd with Fleet Desktop disabled (to get rid of the extra API calls).

Reply:
Too bad we don't have a flag that indicates whether capabilities have been fetched or not. We could check that flag, and, if not set, hit the ping endpoint.